src/main/io/serial.c

```diff
@@ -28,6 +28,8 @@
 #include "config/parameter_group.h"
 #include "config/parameter_group_ids.h"
 
+#include "config/config_master.h"
```

Comment:
Please don't add `config_master.h` now PGs are in it should not be used and will soon disappear.